Simulator/Simulator/Simulator/src/Layer.c

```diff
@@ -293,7 +293,9 @@ Status layer_step_force_spikes(Layer* layer, ArrayBool* spikes, uint32_t time) {
 			neuron_step_force_spike(neuron, time);
```

Comment:
This doen't update the input synapses, this is surely one of the reasons why it doesn't do what i want

Reply:
Fixed